lib/db.js

```diff
@@ -29,7 +29,7 @@ var async = require('async'),
     logger = require('logops'),
     database,
     orionDb,
-    delay = 2000, reportInterval = 15e3, lastErrorTrace = 0;
+    delay = config.checkDB.delay, reportInterval = config.checkDB.reportInterval, lastErrorTrace = 0;
```

Comment:
Do this line comply with our Coding style? I think you should add a line brake after each comma.